src/status_im/subs/wallet/wallet.cljs

```diff
@@ -153,7 +153,8 @@
 (rf/reg-sub
  :wallet/wallet-send-token-symbol
  :<- [:wallet/wallet-send]
- :-> :token-symbol)
+ (fn [{:keys [token-symbol token]}]
+   (or token-symbol (:symbol token))))
```

Comment:
seems like the send flow is initialized with either the `token-symbol` or the `token`, depending on where the flow has started.

Reply:
You are right.

This is because a `token` is a map with balances and `token-symbol` is just the symbol. In some flows we want to send a token but we don't know the account to pick it yet, so we start the flow and the account is picked later.